tests/utils/aws_s3.go

```diff
@@ -0,0 +1,90 @@
+package utils
+
+import (
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func AwsS3CreateFolder(bucketName string, folderPath string) bool {
```

Comment:
it makes sense to return an error in case of failure instead of checking for boolean in the caller